packages/edit-post/src/components/layout/index.js

```diff
@@ -502,8 +501,9 @@ function Layout( {
 					! isEditingTemplate
 						? _templateId
 						: null,
+				// @ZOOMOUTMODE
 				enablePaddingAppender:
-					! isZoomOut() && isRenderingPostOnly && isNotDesignPostType,
+					isRenderingPostOnly && isNotDesignPostType,
```

Comment:
This removal is good, because we never allow the zoom out button when `isRenderingPostOnly` as there isn't a section root id in that context (when we're in post-only mode).